CMakeLists.txt

```diff
@@ -128,6 +128,9 @@ endif()
 if(CORE_SYSTEM_NAME STREQUAL android)
   list(APPEND required_deps Zip)
 endif()
+if(NOT WIN32 AND NOT CORE_SYSTEM_NAME STREQUAL rbpi)
```

Comment:
`... AND NOT CORE_PLATFORM_NAME_LC STREQUAL rbpi`

Reply:
fixed, thx!